bylaws.md

```diff
@@ -24,11 +24,10 @@
 6. In the event of having fewer nominations than vacancies, the vacancy rules will apply.
 
 ## Section 4. Vacancies
-1. When a vacancy arises during the middle of a term, the denhac Members will select by majority vote a replacement director to serve for the remainder of the departing director’s term. In the event of two or more directors departing, the incoming directors will fill the vacated terms. Directors with more votes will fill the vacancies with longer remaining terms respectively.
-2. Nominations for Directors to replace a Director whose term is ending for any reason may come from the Board or from the Members, at a meeting designated expressly for such purpose.
-3. A General Election will be held within 15 days of any event causing the Board of Directors to have 3 or fewer members.
-4. Board members who are considered to have fallen out of good standing shall be dismissed from the board.
-5. Board members may lose their board seat by a supermajority vote of the board, but without affecting their current membership status.
+1. When a vacancy arises during the middle of a term, the denhac Members will select by majority vote a replacement director to serve for the remainder of the departing director’s term. In the event of two or more directors departing, the incoming directors will fill the vacated terms. Directors with more votes will fill the vacancies with longer remaining terms respectively. Vacancies may also be filled as part of the general election process.
```

Comment:
Added `Vacancies may also be filled as part of the general election process.` 

With previous language in bylaws around vacancies requiring their own nomination meeting it was not clear if filling that spot as part of a general election was allowed, required a parallel nomination and election, or was not allowed and the spot would need to be filled later. Adding this sentence clarifies that process.